### DISCUSSION TITLE
posts/llwxj/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#58] [Comments]
# posts/llwxj/

十月底（1031）小王开驴驴去健身，回来路上驴驴突然没反应了，症状就是明明有电但显示屏不亮，拧把也没反应。于是便开启了长达一周多的维修之路，时间线如下：
[&hellip;] 失而复得的感觉很美好，在车店没有以旧换新的原因之一，是因为我还没有做好失去这辆驴驴的准备。现在回想，实体店的衰落不一定是偶然，信息不对称给了贪婪的人可乘之机，代价是信任损失。我笑着和同事说要去小红书写避雷贴，虽然没去写，但我 &hellip;

https://shitao5.org/posts/llwxj/

**Comment by Liam-Zhong** [comment]
赚钱都是信息差嘛，厂子里出货也要赚钱的。但是实体店确实宰得太狠了，还好我不会骑电瓶车😁

**Reply by Shitao5** [reply]
靠信息差、资源整合、创新等方法赚钱都行得通，淘宝/京东的价格也是包含利润的，实体店贵一两百也能理解。如果我遂老板娘的愿以旧换新，相当于她卖出一辆新车的同时，免费得到一辆旧车，她花几块钱修好，还能按照将近千元的价格卖出。懂的人都知道这里面的门道有多深。

天气好的时候骑驴出行其实很舒服，但都需要注意安全。

**Reply by Liam-Zhong** [reply]
确实，像以旧换新这种方式，表面上看是优惠，实际上旧车的价值被低估了，还得再花一大笔买新车。老板“套路”还得靠5老师破～

最近我这里下大雨呢，祝你骑行开心 ：）
